src/entities/purchase-params.ts

```diff
@@ -39,4 +40,15 @@ export interface PurchaseParams {
    * to the RC transaction as metadata.
    */
   metadata?: PurchaseMetadata;
+
+  /**
+   * Defines an optional override for the default branding appearance.
+   *
+   * This property can be used to customize the visual presentation or styling
+   * associated with the branding, replacing the default settings. When set, it
+   * supersedes the pre-configured appearance defined elsewhere in the system.
+   *
+   * @type {BrandingAppearance | undefined}
+   */
+  brandingAppearanceOverride?: BrandingAppearance;
```

Comment:
Hmm I'm not familiar with the need for overriding this in the SDK itself... I wonder if we should mark this as `@internal` if possible, unless we think it's going to be used by developers.

Reply:
Ahhh I just saw the loom recording 😅 . Makes more sense now. I do wonder if we shouldn't extract the UI part into another library that can be used by the SDK + the dashboard actually though... That would avoid us having to expose this API.

Reply:
That would probably be a bigger change though and would make it a bit more difficult to iterate on the SDK... So I'm not totally opposed with going with this option for now I think.